buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -358,6 +375,34 @@ class ClusterFormationTasks {
         return copyPlugins
     }
 
+    /** Configures task to copy a plugin based on a zip file resolved using dependencies for an older version */
+    static Task configureCopyBwcPluginsTask(String name, Project project, Task setup, NodeInfo node) {
+        for (Map.Entry<String, Project> plugin : node.config.plugins.entrySet()) {
+            System.err.println(plugin.getKey() + ": " + plugin.getValue())
+            Project pluginProject = plugin.getValue()
+            if (pluginProject.plugins.hasPlugin(PluginBuildPlugin) == false) {
+                throw new GradleException("Task ${name} cannot project ${pluginProject.path} which is not an esplugin")
+            }
+            String configurationName = "_plugin_bwc_${pluginProject.path}"
```

Comment:
Could the configuration name be calculated using a static help method and use the prefix `_plugin_` here too? So that we can replace:

       if (node.nodeVersion != VersionProperties.elasticsearch) {
            pluginZip = project.configurations.getByName("_plugin_bwc_${plugin.path}")
        } else {
            pluginZip = project.configurations.getByName("_plugin_${plugin.path}")
        }

with 

      FileCollection pluginZip = project.configurations.getByName("_plugin_${plugin.path}")

later?